### DISCUSSION TITLE
How to update Kiwix on android phone

### DISCUSSION THREAD

**Discussion by Beater34** [kiwix/kiwix-android#3622] [Q&A]
Please excuse a newbie question. I have a Moto g54 5g android phone. The latest version of Kiwix on Play Store is 3.8.1, but version 3.9.0 is available as "Direct APK". 3.9.0, which has (I hope) fixed #3606.  How do I update my phone to Kiwix 3.9.0?

**Comment by realeatham** [comment]
Download the direct APK. Open the file. It will say something about the app you are downloading from being in allowed, but one of the buttons will take you to a screen where you can allow it , then open it again.

**Comment by realeatham** [comment]
Sorry for my bad english I am half asleep

**Reply by Beater34** [reply]
Thank you @realeatham for answering my question (in April 2024). I did use the APK in December 2023 to get version 3.9.0 - but anyone needing https://github.com/kiwix/kiwix-android/issues/3606 fixed can now download 3.9.1 from the Play Store (or 3.10.0 for beta testers).

**Comment by kelson42** [comment]
I guess the original question has been answered. Closing.
